### ISSUE TITLE
HTML API documentation

### ISSUE THREAD

**Issue by matt-graham** [mggg/ecological-inference#60]
Submitting as part of JOSS review openjournals/joss-reviews#3397

Although there is API documentation in the docstrings in the code, there is no associated autogenerated HTML documentation which makes it a little difficult to browse. A nice addition would be to have autogenerated API documentation hosted on for example GitHub pages or Read the Docs.

**Comment by pmyteh** [comment]
I would put this as a 'nice-to-have' rather than a blocker for acceptance. But it *would* be nice to have!

**Comment by matt-graham** [comment]
Agreed - this is in reality more a nice-to-have than a blocker for acceptance and I should have made that clearer by not referencing it directly in the review issue - I will update the issue to reflect this.
